sympy/functions/elementary/exponential.py

```diff
@@ -277,9 +278,9 @@ def eval(cls, arg):
 
             coeffs, log_term = [coeff], None
             for term in Mul.make_args(terms):
-                if isinstance(term, log):
```

Comment:
@oscarbenjamin  you can look here-
In this line in sympy master it is getting (log(x)-log(y))(for which `isinstance(term, log)` is false) type of expression so I use logcombine  to check if there are any possibilities to make this type  ` log(exp)` .